test/e2e/framework/ginkgowrapper.go

```diff
@@ -210,12 +210,16 @@ func registerInSuite(ginkgoCall func(string, ...interface{}) bool, args []interf
 			addLabel(fullLabel)
 			if arg.alphaBetaLevel != "" {
 				texts = append(texts, fmt.Sprintf("[%[1]s]", arg.alphaBetaLevel))
-				ginkgoArgs = append(ginkgoArgs, ginkgo.Label("Feature:"+arg.alphaBetaLevel))
+				ginkgoArgs = append(ginkgoArgs, ginkgo.Label(arg.alphaBetaLevel))
 			}
 			if arg.offByDefault {
 				texts = append(texts, "[Feature:OffByDefault]")
-				// TODO: consider this once we have a plan to update the alpha/beta job filters
-				// ginkgoArgs = append(ginkgoArgs, ginkgo.Label("Feature:OffByDefault"))
+				ginkgoArgs = append(ginkgoArgs, ginkgo.Label("Feature:OffByDefault"))
+				// alphas are always off by default but we may want to select
+				// betas based on defaulted-ness
```

Comment:
```suggestion
				// Alphas are always off by default but we may want to select
				// betas based on defaulted-ness.
```

Nit: I prefer proper spelling of full sentences in comments. Makes it easier to add more text later without it starting to look weird.